### DISCUSSION TITLE
如何让demo支持在stackblitz打开

### DISCUSSION THREAD

**Discussion by izkizk8** [umijs/dumi#2140] [Q&A] [answered]
我看有相关功能代码，官网也有，好像是自动生成的，还是自己实现？

**Comment by PeachScript** [comment, marked as answer]
只要按照文档写 demo，就会在 demo 预览框下自动展示对应按钮：https://d.umijs.org/guide/write-demo

**Reply by izkizk8** [reply]
感谢，已经了解了，还可以进一步自定义demo：https://d.umijs.org/config/runtime
建议可以在guide的demo部分说明已集成了代码沙盒，支持demo的简单运行，并且可以自定义修改demo😁
